src/applications/pact-act/containers/questions/burn-pit/BurnPit-2-1.jsx

```diff
@@ -81,11 +78,11 @@ const mapDispatchToProps = {
 
 BurnPit21.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setBurnPit21: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.